### DISCUSSION TITLE
Design Notes - link to UDS Coding Guidebook

### DISCUSSION THREAD

**Discussion by SuzanneHunt-KC** [naccdata/uniform-data-set#64] [Help Needed]
It seems to me that we should provide a link in form instructions that take the user to the NACC webpage where the Coding Guidebook can be downloaded.  However, the text indicating "for Initial Visit Packet" should be dropped from the REDCap form instructions.  We will have just one REDCap form that gets used for all visit types.  I propose that we use the following URL:
https://naccdata.org/data-collection/forms-documentation/uds-3
@mcleod42

**Comment by mcleod42** [comment]
Hi Suzanne - I can bookmark that page in everyone's sandbox projects. I can also add a section to the Developer/Reviewer Guidelines that explains folks can ignore the 'for Initial Visit Packet' instructions.
